tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -337,8 +337,8 @@ def test_list_experiments_paginated_last_page(self):
             returned_experiments.extend(result)
         self.assertEqual(result.token, None)
         # make sure that at least all the experiments created in this test are found
-        returned_exp_id_set = set([exp.experiment_id for exp in returned_experiments])
-        self.assertEqual(set(experiments) - returned_exp_id_set, set())
+        returned_exp_id_set = {exp.experiment_id for exp in returned_experiments}
+        self.assertEqual({experiments} - returned_exp_id_set, set())
```

Comment:
```suggestion
        self.assertEqual(set(experiments) - returned_exp_id_set, set())
```

`{x}` is not equivalent to `set(x)`.